content/blog/2025/06/2025-06-03-birajit-saikia-gsoc-community-bonding-blog-post.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Meet Birajit Saikia GSoC 2025 Contributor Working on Complete Build Retooling of jenkins.io
+tags:
+  - gsoc
+  - gsoc2025
+  - jenkins.io
+  - jenkins
+  - documentation
+  - tools
+  - development
+  - ui
+authors:
+  - biru-codeastromer
+opengraph:
+  image: /images/gsoc/2025/blogpost_birajit.png
+sig:
+  - gsoc
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+Hello everyone! 
+
+I'm Birajit Saikia, and I’m thrilled to be a Google Summer of Code 2025 contributor with Jenkins. This summer, I’ll be working on the project "Complete Build Retooling of jenkins.io", which focuses on modernizing and reorganizing the Jenkins documentation infrastructure and updating the tech stack with UI/UX changes.
```

Comment:
```suggestion
I'm Birajit Saikia, and I’m thrilled to be a Google Summer of Code 2025 contributor with Jenkins. This summer, I’ll be working on the "Complete Build Retooling of jenkins.io" project, which focuses on modernizing and reorganizing the Jenkins documentation infrastructure and updating the tech stack with UI/UX changes.
```
Small suggestion to move the project title so that it makes more sense gramatically.